apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -39,41 +170,130 @@
     content: ' ';
     display: block;
     visibility: hidden;
-    background-color: red;
+    background-color: #D32E6A;
     width: 0;
-    height: 2px;
     transition: width 2.5s;
     transition-timing-function: linear;
+    margin-top: 14px;
   }
 
   &.recording {
     &::after {
       visibility: visible;
       width: 100%;
+      height: 5px;
     }
   }
 }
 
 .recorder-controls {
   display: flex;
+  flex-direction: column;
+
+  .btn {
+    width: 100%;
+    margin-bottom: 0.75em;
+  }
+
+  @media (min-width: $small) {
+    flex-direction: row;
+
+    .btn {
+      order: 1;
+
+      & + .btn {
+        margin-right: 1em;
+        order: 0;
+      }
+    }
+  }
+}
+
+.frame-scroller {
+  overflow-x: auto;
+  overflow-y: hidden;
+  padding: 10px;
 
-  button {
-    flex: 1;
-    margin: 0 10px;
+  @media (min-width: $small) {
+    overflow: visible;
+    padding: 0;
   }
 }
 
 .frame-selector {
+  $frame-width: 150px;
+  $frame-spacing: 20px;
+
   list-style: none;
+  width: ($frame-width + $frame-spacing) * 5;
 
   li {
     display: inline-block;
-    width: 100px;
-    margin: 0 10px;
     cursor: pointer;
+    width: $frame-width;
+    height: $frame-width;
+    margin-right: $frame-spacing;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
+    position: relative;
 
-    img {
-      width: 100%;
+    .selected-indicator {
+      position: absolute;
+      opacity: 0;
+      width: 0;
+      top: 50%;
+      left: 50%;
+      transition: all 0.4s;
+    }
+
+    &.selected {
+      background-color: #ccc;
+
+      .selected-indicator {
+        opacity: 1;
```

Comment:
Properties should be ordered top, left, width, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)